apollo-backend/src/test/java/io/logz/apollo/helpers/StandaloneApollo.java

```diff
@@ -76,6 +81,21 @@ public static StandaloneApollo getOrCreateServer() throws ScriptException, IOExc
         return instance;
     }
 
+    public ApolloApplication createAndStartSlave(List<Integer> environmentIds) {
+        if (instance == null) {
+            throw new RuntimeException("Can't create slave without master first");
+        }
+
+        System.setProperty(SlaveService.SLAVE_PROPERTY, "true");
```

Comment:
The two should be in the configuration, we use hucon you can use ENV vars there.